.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    machine: true
```

Comment:
@tfhartmann - there are official docker images for all recent Terraform versions including 0.9.11 - `hashicorp/terraform:0.9.11`. [See all tags](https://hub.docker.com/r/hashicorp/terraform/tags/). Or do you do something special in your  `fitnesskeeper/terraform`?